### ISSUE TITLE
mupdf is built without clipboard or unicode input support

### ISSUE THREAD

**Issue by vojta001** [NixOS/nixpkgs#191164]
### Describe the bug
`mupdf` is built without clipboard or unicode input support. It prints the following warnings to the console

```
warning: This version of MuPDF has been built WITHOUT clipboard or unicode input support!
warning: Please file a complaint with your friendly local distribution manager.
```

### Steps To Reproduce
1. Run `mupdf anything.pdf` and observe the CLI output.

### Expected behavior
`mupdf` does not output any warnings and supports clipboard

### Notify maintainers
@vrthra @fpletz 

### Metadata

Used mupdf from its last nixpkgs change 48728f5ca4b90407bc83fd02ab5ca7f6ed9d9d93

**Comment by Moredread** [comment]
It looks like that feature needs the thirdparty freeglut:


https://github.com/ArtifexSoftware/thirdparty-freeglut/commit/2ba99eb5264fcc0c3c817dc515aef4b630b419b1#diff-278200a05de757c016a8c1e45e7ad3486638b191f2b0e09e9a6b916c45b28049R123

https://github.com/ArtifexSoftware/mupdf/blob/7cf371e59bcfae528681bb343df65ae83d64472b/platform/gl/gl-ui.c#L449-L453

**Comment by fpletz** [comment]
Fixed by #191202/
